src/ui/util.coffee

```diff
@@ -191,8 +192,19 @@ insertTextAtCursor = (el, text) ->
         range.text = text
         range.select()
 
+# AutoLaunch requires a path unless you are running in electron/nw
+vesrions = process?.versions
+if versions? and (versions.nw? or versions['node-webkit']? or versions.electron?)
+    autoLaunchPath = undefined
+else
+    autoLaunchPath = process.cwd()
```

Comment:
I think that you're correct. That's a more appropriate method. I've updated the code to use `process.execPath`.